crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -88,6 +91,7 @@ pub struct SystemStage {
     last_tick_check: u32,
     /// If true, buffers will be automatically applied at the end of the stage. If false, buffers must be manually applied.
     apply_buffers: bool,
+    must_use_resource: Option<ComponentId>,
```

Comment:
`must_read` and `must_use` are used interchangeably in this file. I think we should sync up terminology. I think I prefer the more explicit `must_read`?